README.md

```diff
@@ -272,6 +296,14 @@ Type: `map(string)`
 
 Default: `null`
 
+### <a name="input_trusted_ca"></a> [trusted\_ca](#input\_trusted\_ca)
+
+Description: Name of the certificate file path for your proxy server.
```

Comment:
nit: I think trustedCa can be both, a string or a file path. We need to update the description accordingly.